analyzers/src/SonarAnalyzer.CSharp/Rules/LoggerMembersNamesShouldComply.cs

```diff
@@ -0,0 +1,90 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2024 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Text.RegularExpressions;
+
+namespace SonarAnalyzer.Rules.CSharp;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public sealed class LoggerMembersNamesShouldComply : ParametrizedDiagnosticAnalyzer
+{
+    private const string DiagnosticId = "S6669";
+    private const string MessageFormat = "Rename this {0} '{1}' to match the regular expression '{2}'.";
+    private const string DefaultFormat = "^_?[Ll]og(ger)?$";
+
+    private static readonly DiagnosticDescriptor Rule = DescriptorFactory.Create(DiagnosticId, MessageFormat, isEnabledByDefault: false);
```

Comment:
Looking at the other parametrized analyzers, we pass `isEnabledByDefault` to false.
Do you know why this is the case?

Reply:
Thanks a lot, very useful info!